test-app/runtime/src/main/cpp/v8_inspector/src/inspector/protocol/CSS.cpp

```diff
@@ -343,6 +347,48 @@ std::unique_ptr<CSSRule> CSSRule::clone() const {
     return parse(serialize().get(), &errors);
 }
 
+std::unique_ptr<RuleUsage> RuleUsage::parse(protocol::Value* value, ErrorSupport* errors) {
+    if (!value || value->type() != protocol::Value::TypeObject) {
```

Comment:
Consider using `!value->isNull()` as IsNull checks for m_type which you're using `value->type()`

Reply:
These are the generated protocol definitions, and I'd rather not modify them, as we'd have to consider updating them each time a change is made to the public protocol.